mothership/holesky-devnet/values.yaml

```diff
@@ -38,3 +38,9 @@ node:
     resources:
       requests:
         memory: 4Gi
+
+bundler:
+  count: 0
+  loadBalancerExternal: false
+  gethRpc: http://node-1:8545
+  entryPointContract: "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789"
```

Comment:
I think it'd be ok to remove these values while it's not being used in devnet now (count: 0)